features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -1288,7 +1300,7 @@ export default function Environments() {
      */
     function getGatewayAccessUrl(vhost, type) {
         const endpoints = { primary: '', secondary: '', combined: '' };
-        if (!vhost && !vhost.host) {
+        if (!vhost || !vhost.host) {
```

Comment:
isn't it `!vhost` is enough?

Reply:
Yes, vhost is not undefined, it should have a host value now. Previously there could be a vhost with null value
fix with https://github.com/wso2/carbon-apimgt/pull/10002/commits/4941c7ab8f49076def9fd81f927305bd9939cf5e